kornia/geometry/warp/homography_warper.py

```diff
@@ -36,6 +112,7 @@ class HomographyWarper(nn.Module):
         align_corners(bool): interpolation flag. Default: False. See
         https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.interpolate for detail
     """
+    _warped_grid: Optional[torch.Tensor]
```

Comment:
why do you define the variable as a class variable. I have the feeling that it is dangerous. why not an instance variable?

Reply:
It is not a class variable, it is just a typing annotation: https://www.python.org/dev/peps/pep-0526/#class-and-instance-variable-annotations

It is required if we want the instance of this class to be torchscriptable. See “Module Attributes” section of https://pytorch.org/docs/stable/jit_language_reference.html#language-reference for details.

It was not the purpose of this PR, but I have made it so in the future it is easier to make the instances of this class torchscriptable.

Reply:
ok, I never did the typing there but if it is a requirement of jit its fine.